dotcom-rendering/src/web/components/EmailSignup.tsx

```diff
@@ -0,0 +1,132 @@
+import { css } from '@emotion/react';
+import {
+	brandAltBackground,
+	from,
+	headline,
+	space,
+	textSans,
+} from '@guardian/source-foundations';
+import { SvgEnvelope } from '@guardian/source-react-components';
+import React from 'react';
+import { SecureSignup } from './SecureSignup';
+
+type Props = {
+	newsletterId: string;
+	name: string;
+	description: string;
+	frequency: string;
+	successText: string;
+};
+
+const Container = ({ children }: { children: React.ReactNode }) => {
+	return (
+		<div
+			css={css`
+				border: black 3px dashed;
+				border-radius: 12px;
+				padding: 10px;
+				margin-bottom: ${space[3]}px;
+			`}
+		>
+			{children}
+		</div>
+	);
+};
+
+const Title = ({ name }: { name: string }) => {
+	return (
+		<div
```

Comment:
Could this be a hX or p tag?

Reply:
good point - I'd better use a p tag - having a hx might mess up the heading hierarchy and make the page less parseable on a screen reader